waspc/packages/deploy/src/providers/fly/cmd/cmd.ts

```diff
@@ -20,16 +20,20 @@ export async function cmd(flyctlArgs: string[], options: CmdOptions): Promise<vo
 	const tomlFilePaths = getTomlFilePaths(options);
 	const commonOps = getCommonOps(options.context, options.waspProjectDir, tomlFilePaths);
 
-	await runFlyctlCommand(commonOps, flyctlArgs);
+	await runFlyctlCommand(commonOps, flyctlArgs, options.org);
```

Comment:
Can we put `org` in `flyctlArgs` before, so we don't need another function param?

Reply:
I think there was a reason I didn't add it, but let me go back and verify why.

Reply:
Good catch, setting it earlier now, thnx 👍🏻